combat-trainer.lic

```diff
@@ -3530,13 +3541,17 @@ class GameState
   end
 
   def wield_whirlwind_offhand
+    return if twohanded_weapon_skill?
     return unless currently_whirlwinding
     @equipment_manager.wield_weapon_offhand(whirlwind_offhand_name)
   end
 
   def determine_whirlwind_action
-    if @use_weak_attacks && Flags['ct-accuracy-ready']
-      update_analyze_array(true) if @analyze_combo_array.empty?
+    if @flame_for_expertise && ( (DRStats.mana < @flame_inner_fire_threshold) || (DRSkill.getxp('Expertise') < 15) )
```

Comment:
To let people do `analyze flame` anytime they're below their preferred inner fire threshold, thoughts on changing this conditional to be:
```suggestion
    if (DRStats.mana < @flame_inner_fire_threshold) || (@flame_for_expertise && DRSkill.getxp('Expertise') < 15)
```
This would then do `analyze flame` if either of these two conditions are true:
1. You're below your IF threshold, or
2. You want to use flame to train expertise and your learning rate is below a threshold

Reply:
Thank you, that's the feedback with system design I really needed, appreciate it.

Reply:
Sure thing, thanks for the quick update. I'm looking forward to using it to keep my IF above 0, haha

Reply:
❤️